swagger.yaml

```diff
@@ -3152,638 +3152,4315 @@ paths:
-        type: number
-        format: double
-        x-nullable: true
-        description: Maximum network average latency (RTT) at L3 for role.
-      packet_loss_percentage:
-        type: number
-        format: double
-        x-nullable: true
-        description: Maximum packet loss allowed at L3 for role.
+      args:
+        description: List of additional arguments passed to coreos-installer
+        example: ["--append-karg", "ip=192.0.2.2::192.0.2.254:255.255.255.0:core0.example.com:enp1s0:none", "--save-partindex", "1", "-n"]
+        type: array
+        items:
+          type: string
 
-  versioned-host-requirements:
+  steps:
     type: object
     properties:
-      version:
+      next_instruction_seconds:
+        type: integer
+      post_step_action:
         type: string
-        description: Version of the component for which requirements are defined
-      master:
-        x-go-name: MasterRequirements
-        $ref: '#/definitions/cluster-host-requirements-details'
-        description: Master node requirements
-      worker:
-        x-go-name: WorkerRequirements
-        $ref: '#/definitions/cluster-host-requirements-details'
-        description: Worker node requirements
-      sno:
-        x-go-name: SNORequirements
-        $ref: '#/definitions/cluster-host-requirements-details'
-        description: Single node OpenShift node requirements
+        description: What to do after finishing to run step instructions
+        default: continue
+        enum:
+        - exit
+        - continue
+      instructions:
+        type: array
+        items:
+          $ref: '#/definitions/step'
+
+
+  step-type:
+    type: string
+    enum:
+      - connectivity-check
+      - execute
+      - inventory
+      - install
+      - free-network-addresses
+      - reset-installation
+      - dhcp-lease-allocate
+      - api-vip-connectivity-check
+      - ntp-synchronizer
+      - installation-disk-speed-check
+      - container-image-availability
+      - domain-resolution
 
-  host-requirements-role:
+  step:
     type: object
     properties:
-      cpu_cores:
-        type: integer
-      ram_gib:
-        type: integer
-      disk_size_gb:
-        type: integer
-      installation_disk_speed_threshold_ms:
-        type: integer
+      step_type:
+        $ref: '#/definitions/step-type'
+      step_id:
+        type: string
+      command:
+        type: string
+      args:
+        type: array
+        items:
+          type: string
 
-  event-list:
+  steps-reply:
     type: array
     items:
-      $ref: '#/definitions/event'
+      $ref: '#/definitions/step-reply'
 
-  event:
+  step-reply:
     type: object
-    required:
-      - cluster_id
-      - severity
-      - message
-      - event_time
     properties:
-      cluster_id:
-        type: string
-        format: uuid
-        description: Unique identifier of the cluster this event relates to.
-        x-go-custom-tag: gorm:"index"
-      host_id:
```

Comment:
it looks like there is one 'event' definition for both /v1 and /v2 cluster events, therefore, changing the field host_id to agent_id (on v1 APIs) break the current API

Reply:
You're right, fixed.